jellyfin.code-workspace

```diff
@@ -1,17 +0,0 @@
-{
-    "folders": [
-        {
-            "path": "."
-        }
-    ],
-    "settings": {
-        "files.eol": "\n",
-        "editor.renderWhitespace": "boundary",
-        "files.exclude": {
-            "**/bin": true,
```

Comment:
I chose not to bring this exclusion over, since the step to copy the web client static files depends on finding the build output directory, which this would hide in the Explorer. In my opinion, the `bin` directories don't add too much noise and hiding them causes some confusion and hunting in the terminal.